src/resources/postcss/views/_loader.pcss

```diff
@@ -0,0 +1,24 @@
+.tribe-events-container {
```

Comment:
this should just be `.tribe-events`

Reply:
On this particular case might be both. We wanted in particular to the Views Manager container to be relative so the loader would cover all of it.